<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeciesEditDialog</class>
 <widget class="QDialog" name="SpeciesEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpeciesEditDialog</string>
  </property>
  <widget class="QGroupBox" name="gbSpecies">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>420</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Species</string>
   </property>
   <widget class="QLabel" name="lbCat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDog">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Dog</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMouse">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbRabbit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Rabbit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>              Cat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDog">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>              Dog</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMouse">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>              Mouse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRabbit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>              Rabbit</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
